editor/editor_plugin.cpp

```diff
@@ -288,6 +296,9 @@ void EditorInterface::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("save_scene"), &EditorInterface::save_scene);
 	ClassDB::bind_method(D_METHOD("save_scene_as", "path", "with_preview"), &EditorInterface::save_scene_as, DEFVAL(true));
+
+	ClassDB::bind_method(D_METHOD("set_main_screen_editor_by_name", "name"), &EditorInterface::set_main_screen_editor_by_name);
```

Comment:
I don't know if the `_by_name` suffix is necessary since the argument is already a String called `name`, so it's relatively explicit.